### PR TITLE
feature: utterances talk

### PR DESCRIPTION
**Pull request by huija**
https://github.com/utterance/utterances

### CODE REVIEW

**Comment by zwbetz-gh** [comment]
Hi @huija - please add a section to the `README.md` on how to use utterances

File: README.md [hunk 1]
```diff
@@ -128,6 +128,27 @@ To reset to a 'first visit' scenario (e.g. for testing), one needs to either bro
...
+3. If your repo is a fork, navigate to its settings tab and confirm the issues feature is turned on.
+
+Secondly,In the site config file set the param `utterances.repo` to enable it.
+
+E.g. for `config.yaml`
+```yaml
+  utterances:
+    repo: zwbetz-gh/zwbetz-gh.github.io
+    issueTerm: title
+    theme: github-light
```
Comment (zwbetz-gh) [inline README.md:147]: Update this, so it's clear the config must be nested under `params`, and use a generic `username` placeholder

```yml
params:
  utterances:
    repo: username/username.github.io
    issueTerm: title
    theme: github-light
```

File: README.md [hunk 2]
```diff
@@ -128,6 +128,27 @@ To reset to a 'first visit' scenario (e.g. for testing), one needs to either bro
 
 Check your browser's help or documentation for details.
 
+## Enable utterances
+
+`utterances` is a lightweight comments widget built on GitHub issues.
+
+Firstly,Choose the repository utterances will connect to.
```
Comment (zwbetz-gh) [inline README.md:135]:
```suggestion
Firstly, choose the repository utterances will connect to:

```

File: README.md [hunk 3]
```diff
@@ -128,6 +128,27 @@ To reset to a 'first visit' scenario (e.g. for testing), one needs to either bro
 
 Check your browser's help or documentation for details.
 
+## Enable utterances
```
Comment (zwbetz-gh) [inline README.md:131]: Add this heading to the _Table of contents_ above please

**Comment by zwbetz-gh** [comment]
@huija - Would you please upload a screenshot of how this looks when enabled on your local environment? 🙂

**Comment by huija** [comment]
light:
image
dark:
image

File: README.md [hunk 4]
```diff
@@ -128,6 +129,27 @@ To reset to a 'first visit' scenario (e.g. for testing), one needs to either bro
...
+3. If your repo is a fork, navigate to its settings tab and confirm the issues feature is turned on.
+
+Secondly,In the site config file set the param `utterances.repo` to enable it.
+
+E.g. for `config.yaml`
+```yaml
+  utterances:
+    repo: zwbetz-gh/zwbetz-gh.github.io
+    issueTerm: title
+    theme: github-light
```
Comment (zwbetz-gh) [inline README.md:148]:
```suggestion
params:
  utterances:
    repo: username/username.github.io
    issueTerm: title
    theme: github-light
```

**Comment by zwbetz-gh** [comment]
Thanks for the light and dark theme screenshots

**Review by zwbetz-gh — APPROVED** [review]
Thanks @huija

**Pull request merged**